package westion.www.exception;




/**
 * 
 * 调用函数的参数数量有误
 * @version 1.0, 2014-3-20
 * @author westion
 * @since JDK1.7
 */
public class ArgumentNumException extends RuntimeException {
	

	public ArgumentNumException(Throwable cause) {
		super(cause);
	}
	
}
